gatsby/best-practices.md

```diff
@@ -0,0 +1,24 @@
+# Gatsby best practices
+
+## Styling and images
+Gatsby provides solutions for a few common styling needs.
+* *Links*: Use [`<Link>`](https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-link/) instead of `<a>` for local links. This takes advantage of Gatsby's page pre-loading and makes changing between pages much quicker. 
+* *Images*: In Wes Bos's Gatsby course, he recommends using [`gatsby-image`](https://www.gatsbyjs.com/plugins/gatsby-image/), however this package has been deprecated. Instead, use [`gatsby-plugin-image`](https://www.gatsbyjs.com/plugins/gatsby-plugin-image).
```

Comment:
I liked your explanation of why to use the `<Link>` component, above. What benefit/s does the suggested plugin give?